web_development_101/introduction_to_web_development.md

```diff
@@ -0,0 +1,107 @@
+If you know why you are here and that you want to learn web development, skip this lesson.
```

Comment:
I think we can remove this line, this lesson is short so It's not a big commitment to go through it and theres a lot of stuff here that everyone should read even if they know what they are after 😄